.dockerignore

```diff
@@ -5,5 +5,11 @@
 **/node_modules
 **/.parcel-cache
 **/parcel-bundle-reports
+**/venv
 **/var
+**/resources/**/generated
```

Comment:
Should we use whitelist instead of blacklist in dockerignore to avoid including unwanted files in the image?
It could be easy to forgot updating the dockerignore file when adding new things.

Reply:
I tried to switch to an allowlist but it seems that when a pattern is negated, the pattern has a high precedence. That would make `**/` wildcards has no effect.

I am going to set up a command to compare the build context with `git ls-files` instead.

Reply:
Updated.